### PR TITLE
feat: set architecture only on registry1 flavor

### PR DESCRIPTION
**Pull request by MxNxPx**

### CODE REVIEW

**Review by mjnagel — DISMISSED** [review]

**Comment by mjnagel** [comment]
I don't think this is overly important but in thinking this through this would result in a weird behavior if someone on an ARM machine attempts to create the registry1 flavor. The result would be a successful build with an empty zarf package (no components). It could cause confusion if they go to deploy it or the test bundle and see nothing happen.

Not sure if we already call this out but it would be good to note somewhere in the readme that the flavor `registry1` is only functional on `amd64` machines due to image restrictions.

**Comment by zachariahmiller** [comment]
> I don't think this is overly important but in thinking this through this would result in a weird behavior if someone on an ARM machine attempts to create the registry1 flavor. The result would be a successful build with an empty zarf package (no components). It could cause confusion if they go to deploy it or the test bundle and see nothing happen.
> 
> Not sure if we already call this out but it would be good to note somewhere in the readme that the flavor `registry1` is only functional on `amd64` machines due to image restrictions.

Agreed thats confusing and should be reflected in the docs. Doing this change was also recommended by @Racer159 and I am in the process of adding it to the gitlab repo.

**Comment by mjnagel** [comment]
If we really wanted to we could do an only registry1/arm64 component that runs an oncreate action to fail the build 😆. Ideally folks are just consuming the published package so it's probably overkill...but it's an option...

**Comment by UncleGedd** [comment]
Y'all go play some video games, it's Saturday! 😘

**Comment by jeff-mccoy** [comment]
> Y'all go play some video games, it's Saturday! 😘

I personally don't have an opinion on when anyone works. You guys will basically never find me between 9 and 11 am during the week or 4-5 pm due to kid things. I agree with the spirit of Gedd's comment, make sure you find your balance and do what makes you happy--but I'll never assume to know your work schedule. For me personally, it for sure isn't 9-5 😆. 

Tl;dr; thanks for reminding us to keep balance Gedd--but at the same time, I don't want anyone to feel guilty for working on something if they choose to, whether that be evenings or weekends. The company doesn't ask you to, but if you would rather work half a friday and a few things saturday, that's cool too.

**Comment by zachariahmiller** [comment]
Just circling back on this since the approval is pretty old...  Are we all still in alignment on this?

A note in the readme would be nice on the amd64 limitation for registry1, but otherwise i think this is good to go.

**Review by zachariahmiller — CHANGES_REQUESTED** [review]
Could you please just add a note about registry1 flavor being amd64 only in the readme? Thanks!

**Review by zachariahmiller — APPROVED** [review]
🚢 🇮🇹
